test/test_dgmulti_1d.jl

```diff
@@ -89,7 +89,7 @@ end
         t = sol.t[end]
         u_ode = sol.u[end]
         du_ode = similar(u_ode)
-        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
+        @test_broken (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
```

Comment:
```suggestion
        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
```
This passes again (although this belongs to the problematic elixir).